tests/test_runners/test_raw.py

```diff
@@ -286,22 +294,24 @@ def test_record_tests_junit_xml(self):
         with tempfile.TemporaryDirectory() as tempdir:
             test_path_file = os.path.join(tempdir, 'tests.xml')
             test_path_file2 = os.path.join(tempdir, 'tests_2.xml')
+            timestamp1 = dateutil.parser.parse("2021-10-05T12:34:00")
             with open(test_path_file, 'w') as f:
                 f.write('\n'.join([
                     '<?xml version="1.0" encoding="UTF-8"?>',
                     '<testsuites name="test_suite_name1" tests="1" failures="0" errors="0" time="10.123">',
-                    '  <testsuite name="test_suite_name2" errors="0" failures="0" skipped="0" timestamp="2021-10-05T12:34:00" time="10.123" tests="1">',   # noqa: E501
+                    f'  <testsuite name="test_suite_name2" errors="0" failures="0" skipped="0" timestamp="{timestamp1}" time="10.123" tests="1">',   # noqa: E501
                     '    <testcase classname="test_class_name" name="test_case_name" time="10.123">',
                     '    </testcase>',
                     '  </testsuite>',
                     '</testsuites>',
                 ]) + '\n')
 
+            timestamp2 = dateutil.parser.parse("2021-10-05T12:34:56")
```

Comment:
nits: test_path_file2 is defined here https://github.com/launchableinc/cli/pull/843/files#diff-ab0fb1f8c2f413fe3f1b4d7cc33b76f99d93d3cac06b353f285bb5546a3ac15fR296
So, could you define the timestamp2 under the `timestamp1` 

Or, could you move the defining `test_path_file2` part to over the defining `timestamp2`?

Reply:
[8547a0a](https://github.com/launchableinc/cli/pull/843/commits/8547a0ad0533c3ebeb97cc6e80c21e80c19d9b03)